xwiki-platform-core/xwiki-platform-edit/xwiki-platform-edit-ui/src/main/resources/XWiki/EditTranslations.fr.xml

```diff
@@ -37,17 +37,25 @@
   <syntaxId>plain/1.0</syntaxId>
   <hidden>true</hidden>
   <content>edit.inplace.close=Fermer
-
 edit.inplace.page.lockFailed=Impossible de verrouiller la page.
 edit.inplace.page.loadFailed=Impossible de charger la page.
 edit.inplace.page.renderFailed=Impossible d'afficher la page.
+edit.inplace.page.translate=Traduire
+edit.inplace.page.translate.hint=Traduire cette page en {0}
+edit.inplace.page.translate.messageBefore=Vous modifiez la traduction {0} de cette page (version initiale). En effet cette page n''a pas encore été traduite en {1} (langue courante).
+edit.inplace.page.translate.messageAfter=Vous êtes en train de modifier la traduction {0}. Débutez en traduisant le titre de la page.
+edit.inplace.actionButtons.loadFailed=Impossible de charger les boutons d'action.
+
+###############################################################################
+## Deprecated
+## Note: each element should be removed when the last branch using it is no longer supported
+###############################################################################
 
 edit.inplace.page.original=Vous modifiez la page originale ({0})
 edit.inplace.page.translation=Vous modifiez la traduction suivante : {0}.
 
-edit.inplace.page.translate=Traduire
-edit.inplace.page.translate.hint=Traduire cette page en : {0}
-
-edit.inplace.actionButtons.loadFailed=Impossible de charger les boutons d'action.</content>
+## Used to indicate where deprecated keys end
+#@deprecatedend
```

Comment:
Sounds like the `deprecatedstart` marker is missing. Probably something to fix on weblate side, but shouldn't be a problem for the translations.

Reply:
See: https://github.com/xwiki/xwiki-platform/blob/b2bbd5fc0fb5f1e730039a2a28c7ab268bad837f/xwiki-platform-core/xwiki-platform-edit/xwiki-platform-edit-ui/src/main/resources/XWiki/EditTranslations.xml#L58

Reply:
Created https://github.com/translate/translate/issues/4278 to fix this.